twistedchecker/checkers/python3.py

```diff
@@ -1,187 +0,0 @@
-"""
-Checker for python3 compatibility issues.
```

Comment:
@glyph Okay, I can revert this (and disable it if running Python 3, where the `ast` module explodes if you try and import code that violates some of these rules).